strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/main/antlr4/org/carlspring/strongbox/nuget/filter/NugetODataFilter.g4

```diff
@@ -92,9 +92,19 @@ NOT
     'not'
 ;
 
+fragment ENDVALUE
+:
+	[a-zA-Z0-9]
+;
+
+fragment MIDDLEVALUE
+:
+    [a-zA-Z_0-9.\-]
+;
+
 VALUE
 :
-    [a-zA-Z_0-9.]+
+	(ENDVALUE|ENDVALUE MIDDLEVALUE* ENDVALUE)
```

Comment:
@grantfar why `ENDVALUE|ENDVALUE` needed? could it be just `ENDVALUE`?

Reply:
I've just noticed that `VALUE` is just a quoted string, and looks like it would be logical if it could be any string, not just the package name. So seems first variant was good and sorry for this confusion.

Reply:
@grantfar can you please get the first variant here? The one that you implemented before `ENDVALUE` and `MIDDLEVALUE`.